field_encrypt.module

```diff
@@ -85,7 +85,6 @@ function field_encrypt_form_alter(&$form, FormStateInterface $form_state, $form_
       );
 
       // We add functions to process the form when it is saved.
-      $form['actions']['submit']['#submit'][] = 'field_encrypt_update_encryption_form_submit';
```

Comment:
No, it is shown as a Drupal message when you update the field storage settings (through Field UI module). Since only users with proper permissions can configure the fields, only those will see the message.

See \Drupal\field_encrypt\EventSubscriber\ConfigSubscriber::onConfigSave() - line 124